app/common/model/src/main/java/io/syndesis/common/model/bulletin/LeveledMessage.java

```diff
@@ -33,7 +33,14 @@
 
     enum Code {
         SYNDESIS000, // Generic message
-        SYNDESIS001 // There are parameter updates for this connection
+        SYNDESIS001, // One or more properties have been updated
+        SYNDESIS002, // One or more properties have been added or removed
+        SYNDESIS003, // Connector has been deleted
+        SYNDESIS004, // Extension has been deleted
+        SYNDESIS005, // Action has been deleted
+        SYNDESIS006, // One or more required properties is not set
+        SYNDESIS007, // Secrets update needed
+        SYNDESIS008, // Validation Error
```

Comment:
Would it make sense to make the message also part of the `Code` enum?

Reply:
It could but UI should not depend on the server message so they can have a much better message than the one we could think about (and maybe have a localized one)

Reply:
We already localize messages in `messages.properties` in `server-runtime`, it contains validation messages. We could have a joint configuration file with all messages used in the server or the UI at some point...

Reply:
And we will... I'm currently working on that and will come up with a desired config file format in a few days. That's why it is so important not to have hardcoded messages coming from the server within particular calls